ic-os/bootloader/README.adoc

```diff
@@ -0,0 +1,14 @@
+= Bootloader
+
+Bootloader is built somewhat similarly to the Ubuntu system, but is
+kept separately (the bootloader is never changed during system upgrades).
+
+The +grub-mkimage+ utility is used to build both the EFI first stage loader
+(built as +/boot/efi/EFI/Boot/bootx64.efi+, to be placed as +/EFI/Boot/bootx64.efi+
+on the +efi+ partition) as well as grub loadable modules
+(built in +/boot/grub/x86_64-efi+, to be placed in under +/x86_64-efi+ on
+the +grub+ partition).
+
+After this is built, the sub-directories +/boot/efi+ and +/boot/grub+ are
+used to build VFAT filesystem images to go into the first two partitions
+of the final disk image.
```

Comment:
Moved from guestos/docs/build.adoc and I deleted the rest of guestos/docs/build.adoc